legend-engine-test-data-generation/src/test/resources/models/relationalModelTestDataGenerationInput.json

```diff
@@ -0,0 +1,641 @@
+{
```

Comment:
Can we change to use pure grammar in the test so we can test and can we also just keep only the required elements for the test. thanks!

Reply:
The reason why i used the input json is that query is needed for now. 
If I just used the  pure grammar, it's hard to get query.